### ISSUE TITLE
would be helpful to know when you've hit the end of a sequence in the view

### ISSUE THREAD

**Issue by adf-ncgr** [legumeinfo/lis_context_viewer#93]
a nice-to-have, especially for dealing with fragmented genomes; sometimes it's not completely obvious whether a track has ended because synteny has been disrupted by a genomic rearrangement event or simply that there were no more genes before you fell off the end of the sequence. You can sort of infer this on the 5'end by inspecting the coordinates of the track and making a judgment call about starting coordinate is close enough to 1 to make it unlikely that there are more genes between where you are and the beginning of the sequence; the other end is even more ambiguous. it might be worth having some sort of beginning of sequence/end of sequence symbols that we could present in the view when appropriate (and perhaps to make one of us feel like a computer scientist, though I'm sure if we used ^/$ for our symbols we'll get complaints).

**Comment by alancleary** [comment]
Indeed, this would be nice, but inferring the ends of a sequence may be difficult and/or prone to error. Perhaps we should add optional sequence end-ponits in the services API. Perhaps we could add it as meta data that the GCV has explicit use for when present.

**Comment by adf-ncgr** [comment]
right, I didn't mean to suggest the code should make inferences. was sort of thinking about whether we could add it to the gene_order helper table somehow and get it almost for free (ie without making changes to the API) but I haven't thought it through much- just wanted to note for future consideration.

**Comment by alancleary** [comment]
Can be implemented as part of #841.
